modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/schema/SqlSchemaManagerImpl.java

```diff
@@ -72,36 +79,73 @@ public SchemaPlus schema(@Nullable String schema) {
     /** {@inheritDoc} */
     @Override
     @NotNull
```

Comment:
- Two spaces between 'requiredVer={}' and 'latestKnownVer'
- Should we use different exceptions for cases when table not found (normal flow, e.g. table dropped) and invalid schema version (breaks internal invariant)